conf/README.md

```diff
@@ -0,0 +1,64 @@
+# Configuration
+
+This folder contains Monty configurations.
+
+## Experiments
+
+The `experiment` folder contains Monty experiment configurations. Most of these experiments are benchmarks and you can learn more about them at [Running Benchmarks](https://thousandbrainsproject.readme.io/docs/running-benchmarks). The experiments in the `experiment/tutorial` folder are used in [Tutorials](https://thousandbrainsproject.readme.io/docs/tutorials).
+
+### Pretraining models
+
+The pretraining configurations are used for running supervised pretraining experiments to generate the models used for follow-on benchmark evaluation experiments. These only need to be rerun if a functional change to the way a learning module learns is introduced. We keep track of version numbers for these, e.g., `ycb_pretrained_v11`.
```

Comment:
```suggestion
The pretraining configurations are used for running supervised pretraining experiments to generate the models used for follow-on benchmark evaluation experiments. These only need to be rerun if a functional change to the way a learning module learns is introduced. We keep track of version numbers for these, e.g., `ycb_pretrained_v11`.

Note that instead of running pretraining, you can also download our pretrained models as outlined in our [getting started guide](https://thousandbrainsproject.readme.io/docs/getting-started#42-download-pretrained-models).
```